server/src/main/java/com/lunchtime/models/MenuItemDish.java

```diff
@@ -28,11 +22,12 @@
     @Column(name = "portion_size", length = 30)
     private String portionSize;
 
+
     @NotNull
     @DecimalMin(message = "Price is not valid",
                            value = "0")
-    @Column(name = "portion_price", length = 20)
-    private Long portionPrice;
+    @Column(name = "portion_price")
+    private BigDecimal portionPrice;
```

Comment:
Why BigDecimal isn't good decision - 10 UAH 2322 cents
![image](https://user-images.githubusercontent.com/6152066/80457132-7bb6b700-8937-11ea-9679-56b588667fed.png)

I'm not sure, but maybe class Currency can help to you https://docs.oracle.com/javase/6/docs/api/java/util/Currency.html

Or you can create special class Currency and validate this field

Reply:
I have changed BigDecimal to Double. Added field currency and validate it with @Currency